pkgs/development/tools/pyre/default.nix

```diff
@@ -52,13 +53,14 @@ in stdenv.mkDerivation {
 
   checkPhase = ''
     make test
+    # ./scripts/run-python-tests.sh # TODO: once typeshed and python bits are added
   '';
 
   # Note that we're not installing the typeshed yet.
   # Improvement for a future version.
   installPhase = ''
     mkdir -p $out/bin
-    cp _build/all/main.native $out/bin/pyre
+    cp _build/all/main.native $out/bin/pyre.bin
```

Comment:
why rename this?

Reply:
See the linked issue, the python client `pyre` expects this binary to be named `pyre.bin`. Currently we aren't installing the python client. Seems to depend on typeshed as you already noted.